doc/dev/cpu-profiler.rst

```diff
@@ -19,25 +19,17 @@ executing the following::
 Compiling Ceph for Profiling
 ============================
 
-To compile Ceph for profiling, first clean everything. :: 
+To compile Ceph for profiling, first clean everything. ::
 
-	make distclean
+    git clean -dfx
 	
-Then, export the following settings so that you can see callgraph output. :: 
+Finally, compile Ceph. ::
 
-	export CFLAGS="-fno-omit-frame-pointer -O2 -g"
-
-Finally, compile Ceph. :: 
-
-	./autogen.sh
-	./configure
-	make
-
-You can use ``make -j`` to execute multiple jobs depending upon your system. For
-example::
-
-	make -j4
+	./do-cmake.sh -DCMAKE_CXX_FLAGS="-fno-omit-frame-pointer -O2 -g"
+    cd build
+    cmake --build .
 
+In this command, ``CMAKE_CXX_FLAGS`` is specified. This provides callgraph output.can see callgraph output.
```

Comment:
> In this command, ``CMAKE_CXX_FLAGS`` is specified. This provides callgraph output.can see callgraph output.

This looks like a couple of sentences that have been run together accidentally.

Reply:
thank you for your review! ahh, very sorry! i managed to create a melange of the revised and the unreivsed versions!

@zdover23 fixed and repushed again!

Reply:
LGTM